### ISSUE TITLE
svelte version dependency in 0.7.3

### ISSUE THREAD

**Issue by don-wang** [svelteuidev/svelteui#205]
### What package has an issue

@svelteuidev/core

### A clear and concise description of what the bug is

> the latest dev release seems to explicitly state svelte 3.47.0 as the peer dependency.

svelte version dependency #186
The same issue had been fixed and closed, but I got the same error in the latest 0.7.3 ver.

### Steps To Reproduce

In package.json, using `"svelte": "3.49.0"` as `devDependencies`, `"@svelteuidev/core": "^0.7.3"`, as `dependencies`

by run
`npm install`

got error: 
```
npm ERR! Could not resolve dependency:
npm ERR! peer svelte@"3.47.0" from @svelteuidev/core@0.7.3
npm ERR! node_modules/@svelteuidev/core
npm ERR!   @svelteuidev/core@"^0.7.3" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: svelte@3.47.0
npm ERR! node_modules/svelte
npm ERR!   peer svelte@"3.47.0" from @svelteuidev/core@0.7.3
npm ERR!   node_modules/@svelteuidev/core
npm ERR!     @svelteuidev/core@"^0.7.3" from the root project
```

### Do you know how to fix the issue

No

### Are you willing to participate in fixing this issue and create a pull request with the fix

Yes

**Comment by mrbrianevans** [comment]
It would be really nice to update Svelte to the latest version because it seems there is a problem with v3.47.0:
 ## Svelte before 3.49.0 vulnerable to XSS when using objects during server-side rendering
> The package svelte before 3.49.0 is vulnerable to Cross-site Scripting (XSS) due to improper input sanitization and to improper escape of attributes when using objects during SSR (Server-Side Rendering). Exploiting this vulnerability is possible via objects with a custom toString() function.

See [CVE-2022-25875](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-25875)

Projects that use svelteui and have the peer dependency svelte@v3.47.0 are getting dependabot alerts.

**Comment by luisjakon** [comment]
Hi, Is there a solution or workaround in the works for this issue? 
This makes the current state of svelte-ui unfortunately unusable for newer projects...
Many thnxs!

**Comment by don-wang** [comment]
@luisjakon 
I used `npm install --legacy-peer-deps` as temporary solution

**Comment by BeeMargarida** [comment]
This should already be resolved in the new release 0.7.5 (an announcement will be made)

**Comment by don-wang** [comment]
@BeeMargarida 
Sorry, I got the same error in 0.7.5
```npm ERR! Could not resolve dependency:
npm ERR! peer svelte@"3.47.0" from @svelteuidev/core@0.7.5
```

**Comment by BeeMargarida** [comment]
Should be fixed in the new version 0.7.6

**Comment by don-wang** [comment]
@BeeMargarida 

Thank you for your work.
Unfortunately, it seems the '>=' is still being stripped as [Brisklemonade metioned](https://github.com/svelteuidev/svelteui/pull/212#issuecomment-1287341489)

I got this error when run `npm install`
```
npm ERR! Could not resolve dependency:
npm ERR! peer svelte@"3.47.0" from @svelteuidev/core@0.7.6
npm ERR! node_modules/@svelteuidev/core
npm ERR!   @svelteuidev/core@"^0.7.6" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: svelte@3.47.0
npm ERR! node_modules/svelte
npm ERR!   peer svelte@"3.47.0" from @svelteuidev/core@0.7.6
npm ERR!   node_modules/@svelteuidev/core
npm ERR!     @svelteuidev/core@"^0.7.6" from the root project
npm ERR! 
```

It works with `npm install --legacy-peer-deps`, then  in the  `node_modules/@svelteuidev/core/package.json`, it shows
```
  "peerDependencies": {
    "@svelteuidev/composables": "0.7.6",
    "svelte": "3.47.0"
  },
```

Same as in `node_modules/@svelteuidev/preprocessors/package.json`

**Comment by BeeMargarida** [comment]
Hum, >= is in the generated package folder is showing with `>=`. Gotta investigate this

**Comment by JeromeK13** [comment]
Hey Guys

Having the same problem running node v18.12.1

**Comment by jedfonner** [comment]
Same with a version 3.54.0 of svelte and svelteui 0.7.6. Using `--legacy-peer-deps` allowed me to workaround the problem.
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: dead-to-me-list@0.0.1
npm ERR! Found: svelte@3.54.0
npm ERR! node_modules/svelte
npm ERR!   dev svelte@"^3.54.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer svelte@"3.47.0" from @svelteuidev/core@0.7.6
npm ERR! node_modules/@svelteuidev/core
npm ERR!   @svelteuidev/core@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Comment by BeeMargarida** [comment]
https://github.com/svelteuidev/svelteui/pull/246 should fix it, I'll make a new patch release soon

**Comment by BeeMargarida** [comment]
Fixed in new release 0.7.7
